lib/router.js

```diff
@@ -79,9 +79,10 @@ Cherrytree.prototype.map = function (routes) {
 
   function eachBranch (node, memo, fn) {
     node.routes.forEach(function (route) {
-      if (!route.routes || route.routes.length === 0) {
+      if (!route.options || route.options.abstract !== true) {
```

Comment:
The only case for `abstract` is when listing routes to clearly indicate a level is not routable. Say `/account` defaults to `/account/profile`, then abstract could indicate that. However that could also be accomplished by registering each leaf with explict paths, like `path: "account/profile" `.

I like the idea of removing it - I'll do that.